src/core/pyproxy.js

```diff
@@ -214,10 +212,26 @@ Module.pyproxy_destroy = function (proxy, destroyed_msg) {
   }
   let ptrobj = _getPtr(proxy);
   Module.finalizationRegistry.unregister(proxy);
+  destroyed_msg ||= "Object has already been destroyed";
```

Comment:
It looks like `||=` operator is not supported in [Node < 15.0](https://nodejs.org/en/blog/release/v15.0.0/)